src/libraries/System.Private.CoreLib/src/System/Byte.cs

```diff
@@ -301,7 +301,7 @@ object IConvertible.ToType(Type type, IFormatProvider? provider)
         static byte IAdditionOperators<byte, byte, byte>.operator +(byte left, byte right) => (byte)(left + right);
 
         // /// <inheritdoc cref="IAdditionOperators{TSelf, TOther, TResult}.op_CheckedAddition(TSelf, TOther)" />
-        // static byte IAdditionOperators<byte, byte, byte>.operator checked +(byte left, byte right) => checked((byte)(left + right));
+        static byte IAdditionOperators<byte, byte, byte>.operator checked +(byte left, byte right) => checked((byte)(left + right));
```

Comment:
Shouldn't we specify visibility here?
Or is that not good practice for interface implementations?

I haven't worked with interface operators yet :D.

Reply:
These are explicitly implemented and so accessibility modifiers aren't allowed